R/authentication.R

```diff
@@ -11,12 +11,14 @@
 #' @param password the password for your APEX Clearing account.
 #' @return a global environment variable called APEX_token which is used in the other helper functions.
 #' @examples
+#' \dontrun{
 #' get_APEX_auth_token(username = "username", password = "password")
 #' get_APEX_auth_token() # RStudio prompts the user for their login
+#' }
 get_APEX_auth_token <- function(username, password){
   if(missing(username) | missing(password)){
-    username <- rstudioapi::askForSecret("APEX Username")
-    password <- rstudioapi::askForSecret("APEX Password")
+    username <- askForSecret("APEX Username")
+    password <- askForSecret("APEX Password")
```

Comment:
It's probably worth considering a change in the future where this checks whether a session is interactive, and if it isn't it checks `Sys.getenv()` for the username and password.